pennylane/transforms/qcut.py

```diff
@@ -939,6 +940,142 @@ def qcut_processing_fn_mc(
     return qml.math.convert_like(np.mean(expvals), res0)
 
 
+@batch_transform
+def cut_circuit_mc(
+    tape: QuantumTape,
+    shots: int,
+    device_wires: Wires,
```

Comment:
Thanks for changing this to be non-default.

Although now I'm not sure:
- If you use `cut_circuit_mc()` as a transform applied to tapes, then `device_wires` shouldn't be (and isn't) optional
- If you use `cut_circuit_mc()` as a transform applied to a QNode, then `device_wires` should be thought of as optional/not needed at all, because we want to infer the wires from the provided device.

A similar argument to above applies for `shots`.

I think this raises a wider question: _should we write docstrings for batch transforms tailored towards tape transforming or QNode transforming_?

Wonder if @josh146 has any thoughts here?

Reply:
> I think this raises a wider question: should we write docstrings for batch transforms tailored towards tape transforming or QNode transforming?

This is a really good question 😬  Attempting to create an API that targets two different objects, with different behaviour, is nice for developers but has this downside for users/documentation.

I don't have a good solution atm, but my thinking so far has been:

- Be clear on all options in the `Args:` section, but prioritize the QNode approach.

- In `**Example**`, focus on the QNode approach.

- Leave tape use-cases to `UsageDetails`